scripts/status_map.js

```diff
@@ -33,22 +28,6 @@ let shieldImageURLs = Object.fromEntries(
 );
 let shields = ShieldDef.loadShields(shieldImageURLs);
 
-let usSVG = fs.readFileSync(`${process.cwd()}/scripts/blank_map_us.svg`, {
-  encoding: "utf8",
-});
-usSVG = fillPaths(
-  usSVG,
-  Object.keys(shields)
-    .map((network) => network.match(/^US:(\w\w)(?::|$)/))
-    .filter((m) => m)
-    .map((m) => m[1])
-);
-usSVG = usSVG.replace(
-  /<title>.+?<\/title>/,
-  "<title>U.S. states with shields supported by OpenStreetMap Americana</title>"
-);
-fs.writeFileSync(`${process.cwd()}/doc-img/shield_map_us.svg`, usSVG);
-
```

Comment:
Is this code removed because we no longer need a US map?

Reply:
Yep. With Puerto Rico added, the US map is complete so it's no longer necessary.